code/modules/antagonists/traitor/objectives/destroy_item.dm

```diff
@@ -4,7 +4,7 @@
 
 	progression_minimum = 20 MINUTES
 	progression_reward = 5 MINUTES
-	telecrystal_reward = list(2, 4)
+	telecrystal_reward = 0
```

Comment:
This is only used as a subtype, right?

Reply:
yes, most subtypes override it but it should be set to 0 by default